src/planning/src/adapter_planning/eufs.cpp

```diff
@@ -33,4 +43,54 @@ void EufsAdapter::set_mission_state(int mission, int state) {
   }
 }
 
+void EufsAdapter::pose_callback(const eufs_msgs::msg::CarState& msg) {
+  custom_interfaces::msg::VehicleState pose;
+  // only gets the x, y, and theta since those are the only ones necessary for planning
+  pose.position.x = msg.pose.pose.position.x;
+  pose.position.y = msg.pose.pose.position.y;
+  pose.theta = atan2(2.0f * (msg.pose.pose.orientation.w * msg.pose.pose.orientation.z +
+                             msg.pose.pose.orientation.x * msg.pose.pose.orientation.y),
+                     msg.pose.pose.orientation.w * msg.pose.pose.orientation.w +
+                         msg.pose.pose.orientation.x * msg.pose.pose.orientation.x -
+                         msg.pose.pose.orientation.y * msg.pose.pose.orientation.y -
+                         msg.pose.pose.orientation.z * msg.pose.pose.orientation.z);
+  this->node->vehicle_localization_callback(pose);
+}
+
 void EufsAdapter::finish() { std::cout << "Finish undefined for Eufs\n"; }
+
+void EufsAdapter::map_callback(const eufs_msgs::msg::ConeArrayWithCovariance& msg) {
+  RCLCPP_INFO(this->node->get_logger(), "Received cones from EUFS");
```

Comment:
Probably not a good idea to keep this as info, very high frequency, will fill the terminal up